README.md

```diff
@@ -1,2 +1,62 @@
-# ttax
-Tensor-Train toolbox in Jax
+﻿# TTAX: Tensor-Train toolbox on Jax
+Implementation of Tensor-Train toolbox in Jax containing several routines for working with tensors in TT format in Python.
+## Installation
+[JAX](https://github.com/google/jax/blob/master/README.md) and [Flax](https://github.com/google/flax#readme) are required for installation (see instructions there).
+Install TTAX from PyPi:
+```
+> pip install ttax
+```
+## Quick start
+This is a quick starting guide to look at the basics of working with ttax library. Our library provides routines for Tensor-Train object – a compact (factorized) representation of a tensor.
+
+In example below we create TT-tensor, multiply it by constant and convert it to full tensor format.
+```python
+!pip install ttax jax flax
+
+import ttax
+import numpy as np
+
+rng = jax.random.PRNGKey(42)
+t = ttax.random.tensor(rng, [10, 5, 2], tt_rank=3)
+print(ttax.full(2 * t))
+```
+## Structure overview
```

Comment:
The structure overview is hard to read and is unnessary for most users. I would a) move it into some other documentation page b) improve it if possible